xqf.desktop.in

```diff
@@ -1,10 +1,10 @@
 [Desktop Entry]
-_Name=XQF Game Server Browser
-_GenericName=Game Server Browser
-_Comment=Locate and connect to game servers
+Name=XQF Game Server Browser
+GenericName=Game Server Browser
+Comment=Locate and connect to game servers
```

Comment:
msgfmt is only able to process Name GenericName and Comment variables, nothing else.